lib/stink_bomb.rb

```diff
@@ -27,17 +28,25 @@ def reset
     def bombs
       configuration.bombs
     end
-  end
 
-private
+  private
+
+    def sub_message(deadline, message)
```

Comment:
maybe `fill_in_message`?  I was confused about `sub` until I saw the implementation.